src/TurbulenceClosures/turbulence_closure_implementations/CATKEVerticalDiffusivities/CATKEVerticalDiffusivities.jl

```diff
@@ -364,6 +364,7 @@ end
 
 @inline function turbulent_velocityᶜᶜᶜ(i, j, k, grid, closure, e)
     eᵢ = @inbounds e[i, j, k]
+    #eᵢ = ℑxyᶜᶜᵃ(i, j, k, grid, ℑxyᶠᶠᵃ, e)
```

Comment:
this is something you were playing around with?

Reply:
Yes I can delete that